### PR TITLE
feat(provider/kubernetes): Deploy baked docker images

### PR DESCRIPTION
**Pull request by brujoand**
The objective here is to be able to deploy baked docker containers to Kubernetes, so we don't have to rely on using the docker registry trigger, and we get to keep the traceability. 

I haven't synced with anyone on this approach, so I expect to have to alter worthing and/or some of the flow for this. So feel free to make suggestions for change. 

This PR depends on a change to orca: https://github.com/spinnaker/orca/pull/1299

### CODE REVIEW

File: app/scripts/modules/kubernetes/cluster/configure/CommandBuilder.js [hunk 1]
```diff
@@ -137,8 +137,10 @@ module.exports = angular.module('spinnaker.kubernetes.clusterCommandBuilder.serv
     }
 
     function buildImageId(image) {
-      if (image.fromContext) {
+      if (image.fromContext && image.pattern && image.cluster) {
```
Comment (lwander) [inline app/scripts/modules/kubernetes/cluster/configure/CommandBuilder.js]: Does this mean it came from a findImage? I think we should be more explicit here

File: app/scripts/modules/kubernetes/cluster/configure/CommandBuilder.js [hunk 2]
```diff
@@ -189,11 +191,19 @@ module.exports = angular.module('spinnaker.kubernetes.clusterCommandBuilder.serv
           repository: current.name,
           stageId: current.refId
         });
+      } else if (current.type === 'bake') {
+        result.push({
+          fromContext: true,
+          repository: current.ami_name,
+          organization: current.organization,
+          stageId: current.refId,
+          registry: 'containers.schibsted.io'
```
Comment (lwander) [inline app/scripts/modules/kubernetes/cluster/configure/CommandBuilder.js]: Is this meant to be hardcoded here?

File: app/scripts/modules/kubernetes/cluster/configure/CommandBuilder.js (hunk 1 above)
Comment (lwander) [inline app/scripts/modules/kubernetes/cluster/configure/CommandBuilder.js]: e.g. 

```js
if (image.fromContext) {
  if (image.fromBake) {
  } else if (image.fromFindImage) {
  } else {
  // uh oh
  }
} else if (image.fromTrigger && !image.tag) {
}
```

**Comment by lwander** [comment]
Also could you double check this for me @danielpeach?

File: app/scripts/modules/kubernetes/cluster/configure/CommandBuilder.js (hunk 2 above)
Comment (brujoand) [inline app/scripts/modules/kubernetes/cluster/configure/CommandBuilder.js]: Oh, no that was me testing. I'll get that out 👍

File: app/scripts/modules/kubernetes/cluster/configure/CommandBuilder.js (hunk 1 above)
Comment (brujoand) [inline app/scripts/modules/kubernetes/cluster/configure/CommandBuilder.js]: Yeah it's meant to cover the findImage case. I agree, it's good to be clear here.

**Comment by brujoand** [comment]
Thanks guys, I've addressed the comments and gotten back in sync with master.

**Pull request merged**